docs/data-storage.md

```diff
@@ -56,7 +56,7 @@ These databases are
    state this database is never read by the node, only by external services such
    as the wallet-proxy.
 
-The **tree state** is stored in a single LMDB database with 4
+The **tree state** is stored in an LMDB database with 4
 **stores** corresponding to the 4 different items that are stored. LMDB is a
```

Comment:
This still says `4`, but below it says five as it should with the addition of metadata.

Reply:
Fixed.